src/components/NotificationTypeSelector/index.tsx

```diff
@@ -5,15 +5,17 @@ import NotificationType from './NotificationType';
 const messages = defineMessages({
   mediarequested: 'Media Requested',
   mediarequestedDescription:
-    'Sends a notification when new media is requested. For certain agents, this will only send the notification to admins or users with the "Manage Requests" permission.',
+    'Sends a notification when media is requested and requires approval.',
   mediaapproved: 'Media Approved',
-  mediaapprovedDescription: 'Sends a notification when media is approved.',
+  mediaapprovedDescription:
+    'Sends a notification when media is approved.\
+    By default, automatically approved requests will not trigger notifications.',
```

Comment:
What's happening here?

Reply:
We get a lot of questions from users about why they aren't receiving notifications, so I wanted to clarify that the default behavior is that auto-approved requests don't trigger notifications.  Perhaps I need to rethink the wording here?

Reply:
No, I mean what's up with the slash in the middle of it? Haha

Reply:
Oh it's just to break it into multiple lines for readability.